crates/egui/src/widgets/color_picker.rs

```diff
@@ -311,6 +314,51 @@ fn color_picker_hsvag_2d(ui: &mut Ui, hsva: &mut HsvaGamma, alpha: Alpha) {
     color_slider_2d(ui, v, s, |v, s| HsvaGamma { s, v, ..opaque }.into());
 }
 
+fn srgba_edit_ui(ui: &mut Ui, hsvag: HsvaGamma) -> Option<HsvaGamma> {
```

Comment:
This function should take `alpha: Alpha` as argument and not show an alpha-slider for `Alpha::Opaque`, for instance.

Reply:
Added alpha: Alpha parameter, alpha-slider won't be shown when Alpha::Opaque